### DISCUSSION TITLE
How to create footer column

### DISCUSSION THREAD

**Discussion by goodlythink** [icflorescu/mantine-datatable#50] [Q&A]
![Screen Shot 2565-09-22 at 10 00 16](https://user-images.githubusercontent.com/12431734/191648797-ead88682-e06d-46bf-b90a-96b6fa88b11a.png)

Will it be possible? If adding columns at the end of the table and merge columns

**Comment by icflorescu** [comment]
No, I'm afraid not.

**Reply by goodlythink** [reply]
thank
